app/src/main/java/org/inspire/breath/data/DbExamples.java

```diff
@@ -0,0 +1,62 @@
+package org.inspire.breath.data;
+
+import android.content.Context;
+import android.widget.Toast;
+
+import java.util.List;
+
+/*
+    This class is intended to demonstrate interaction with the database,
+    *not* to be used as a helper
+    An instance of the database exists in LoginActivity.java, so should be
+    accessible with LoginActivity.db
+    For testing purposes, queries are allowed on main thread. They will need
+    to run in a worker thread for production, so don't push code with queries
+    running on the main thread.
+ */
+public class DbExamples {
```

Comment:
Do we need an examples class?

Reply:
If the rest of the team find it helpful, otherwise not really. If we want to lose it I might keep the resetDB function to aid testing

Reply:
I think it is useful but I think it belongs in more of a wiki than a codebase if that makes sense? I've repurposed the VSTS repo to be a wiki of sorts :) so if you clone from here:
`git clone git@ssh.dev.azure.com:v3/WHELANB7/Breathoscope/Breathoscope`
you can put a little markdown guide under docs, commit it and push it to the origin then it should show up over [here](https://dev.azure.com/WHELANB7/Breathoscope/_wiki/wikis/Breathoscope%20-%20Code?wikiVersion=GBmaster&pagePath=%2FREADME&pageId=2)

This way we can keep the useful example code and keep our code base clean :)

Reply:
Yeah I had considered that, didn't know we were using the other repo - definitely the better option. I'll just remove this now for the purpose of getting this merged and will push to the other repo later

Reply:
Cool, thanks. I only just repurposed the repo like 10 minutes before I wrote that btw :P specifically so we can have this sort of handy info :)